src/corg/development-tools-and-process/git-usage.md

```diff
@@ -545,103 +545,105 @@ Prerequisites: having Git installed. For installation and configuration instruct
 To **clone a repository**, do:
 
 ```
-$ git clone git://github.com/nuxeo/nuxeo.git
+git clone git://github.com/nuxeo/nuxeo.git
 
 ```
 
 To **retrieve remote changes**, do:
 
 ```
-$ git fetch
+git fetch
 
 ```
 
 To **update your repository according to remote changes**, do:
 
 ```
-$ git pull --rebase
+git pull --rebase
 
 ```
 
 This is equivalent to:
 
 ```
-$ git fetch && git rebase
+git fetch && git rebase
 
 ```
 
 To **view the current status** of your repository, do:
 
 ```
-$ git status
+git status
 
 ```
 
-To commit local changes to a file, do:
+To **commit local changes** to a file, do:
 
 ```
-$ git commit -m "my commit message" myfile.txt
+git commit -m "NXP-xxxx: my commit message" myfile.txt
 
 ```
 
-or, if you'd like to handle all modified files at once:
+or, to **commit all modified files** at once:
 
 ```
-$ git commit -m "my commit message" .
+git commit -m "NXP-xxxx: my commit message" -a
```

Comment:
Because the message content `"my commit message"` splits the command into two pieces, so I would prefer to keep the short argument(s) before a long one. It's just my own opinion.